RNWCPP/PropertySheets/React.Cpp.props

```diff
@@ -11,6 +11,7 @@
 
   <PropertyGroup Label="Configuration">
     <PlatformToolset>v141</PlatformToolset>
+    <PlatformToolset Condition="'$(VisualStudioVersion)' == '16.0'">v142</PlatformToolset>
```

Comment:
Is this wise?
Aren't we using a v141-equivalent internally?

Note, VS2019 can still use the v141 toolset.

Reply:
What do you mean by that?  yes when building with 2017  - which is what our current cloud builds do, everything is v141.  I'm assuming when we move up to VS2019 and v142 tools in devmain we'll want to match that . (though v141 & v142 outputs are supposed to have binary compatibility)

Reply:
Meaning, maybe we should target the same toolset version regardles of the Visual Studio IDE version installed.

Reply:
Default projects created by VS2019 have the line I am adding, if using 2019 it targets v142.